docs/guides/recipe.rst

```diff
@@ -69,21 +74,52 @@ non-executable fields of ``package.json`` such as ``name``, ``version``,
 * **keywords**: A list of strings that are related to the package.
 * **maintainers**: A list of ``Maintainer`` objects that describe the contact
   information of the people responsible for the recipe (you, in this case).
-* **home**: ?
-* **sources**: ?
-* **icon**: A publicly-accessible URI to give the recipe a friendly face.
+* **home**: The homepage for this Recipe.
+* **sources**: A list of necessary container image URIs.
```

Comment:
This field is similar to Python's extra links - it can be the link to the container image but it can also be link to documentation or source code and so on.